bin/MadGraph5_aMCatNLO/cards/production/13TeV/wellnu012j_5f_NLO_FXFX_bias_binned/wellnu012j_5f_NLO_FXFX_bias_binned_madspin_card.dat

```diff
@@ -0,0 +1,37 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ #set BW_cut 15                # cut on how far the particle can be off-shell
```

Comment:
please remove commented instructions for better clarity

Reply:
is madspin really needed if no intermediate resonance is used?!? (@kdlong, @qliphy)